scripts/launch/connect-to-cluster/default.nix

```diff
@@ -84,6 +84,6 @@ in pkgs.writeScript "${executable}-connect-to-${environment}" ''
     ${ ifWallet "--wallet-db-path '${stateDir}/wallet-db'"}        \
     --keyfile ${stateDir}/secret.key                               \
         ${ ifWallet "--wallet-address ${walletListen}"}    \
-        --ekg-server 127.0.0.1:8080 --metrics +RTS -T -RTS             
+        --ekg-server 0.0.0.0:8080 --metrics +RTS -T -RTS             
```

Comment:
Hmm, actually we should default to 127.0.0.1 and make this overridable (which would be set to 0.0.0.0 for docker, same as for walletListen).

Reply:
Is `ekgServerListen` good?

Reply:
Yup!